tests/simpleKVBC/TesterReplica/main.cpp

```diff
@@ -53,6 +54,8 @@ void run_replica(int argc, char** argv) {
   if (!setup->GetReplicaConfig().isReadOnly) replica->setReplicaStateSync(new ReplicaStateSyncImp(blockMetadata));
 
   auto cmdHandler = std::make_unique<InternalCommandsHandler>(replica.get(), replica.get(), blockMetadata, logger);
+  cmdHandler->setPruningHandler(std::make_shared<concord::reconfiguration::pruning::PruningHandler>(
```

Comment:
We need to add adding the `concord::reconfiguration::pruning::PruningHandler` as a default one in a Dispacher's c-r as we add the default Reconfiguration handler